### PR TITLE
Add new webserver_config_params

### PR DESCRIPTION
**Pull request by kazukifujii**
Since there was no item to be set during performance tuning, it was added.

### CODE REVIEW

File: libraries/drivers_appserver_passenger.rb [hunk 1]
```diff
@@ -5,7 +5,15 @@ module Appserver
     class Passenger < Drivers::Appserver::Base
       adapter :passenger
       allowed_engines :passenger
-      output filter: %i[max_pool_size min_instances mount_point]
+      WEBSERVER_CONFIG_PARAMS = %i[
+        max_pool_size min_instances
```
Comment (ajgon) [inline libraries/drivers_appserver_passenger.rb]: Please, move it to new line.

File: spec/unit/recipes/configure_spec.rb [hunk 2]
```diff
@@ -872,6 +872,10 @@
         deploy['dummy_project']['appserver']['max_pool_size'] = 10
         deploy['dummy_project']['appserver']['min_instances'] = 5
         deploy['dummy_project']['appserver']['mount_point'] = '/some/mount/point'
+        deploy['dummy_project']['appserver']['pool_idle_time'] = 300
+        deploy['dummy_project']['appserver']['max_request_queue_size'] = 100
+        deploy['dummy_project']['appserver']['error_document'] = { "503": '503.html', "504": '504.html' }
+        deploy['dummy_project']['appserver']['passenger_max_preloader_idle_time'] = 300
```
Comment (ajgon) [inline spec/unit/recipes/configure_spec.rb]: Please, add extra tests to `it 'creates apache2 passenger config'` spec, to test if those params are actually reflected in config.

Comment (kazukifujii) [reply spec/unit/recipes/configure_spec.rb]: Added tests to it 'creates apache2 passenger config'.

**Comment by kazukifujii** [comment]
Thank you for the review! Added and fixed missing cord.

**Pull request merged**